### PR TITLE
Campfire/sort fixup

### PR DESCRIPTION
**Pull request by aswanson-nr**
## Description
Quick PR to clean up some git stuff going on in #1945

This adds a sort component which sets a sort value in the query params and navigates to that page. The main IO page then reloads and queries with the sort value from the params

Screen Shot 2021-12-13 at 3 57 57 PM

### CODE REVIEW

File: src/components/quickstarts/QuickstartSort.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const QuickstartSort = ({ className }) => {
+  const params = new URLSearchParams(location.search);
+  const sortParam = params.get('sort');
+  const handleChange = (event) => {
+    params.set('sort', event.target.value);
+    navigate(`/instant-observability?${params.toString()}`);
+  };
+  return (
+    <>
```
Comment (aswanson-nr) [inline src/components/quickstarts/QuickstartSort.js:13]: Looks like we don't need this fragment!

**Review by aswanson-nr — COMMENTED** [review]
Noticed an issue where `location` is available during render if you go directly to `/instant-observability` on a prod build. 
Screen Shot 2021-12-14 at 2 44 00 PM

Also it would be worth replicating the mobile tab dropdown styling for the sort select.

File: src/components/quickstarts/QuickstartSort.js (hunk 1 above)
Comment (zstix) [reply src/components/quickstarts/QuickstartSort.js:14]: Looks like this comment got prematurely resolved.

**Review by zstix — APPROVED** [review]
Looks good. There are some areas that we might want to consider refactoring, but that doesn't need to happen in this PR.

**Pull request merged**